src/foam/u2/Element.js

```diff
@@ -352,7 +365,7 @@ foam.CLASS({
       this.state = this.LOADED;
       if ( this.tabIndex ) this.setAttribute('tabindex', this.tabIndex);
       // Add a delay before setting the focus in case the DOM isn't visible yet.
-      if ( this.focused ) window.setTimeout(() => { try { this.el().focus(); } catch(x) {} }, 70);
+      if ( this.focused ) window.setTimeout(() => { try { this.el_().focus(); } catch(x) {} }, 70);
```

Comment:
I think the setTimeout was to protect against el() not being ready yet. If we do el().then(), then I think we can get rid of the setTImeout.